pkg/query/selection.go

```diff
@@ -88,6 +91,14 @@ const (
 	FieldQuery CriterionType = "fieldQuery"
 	// LabelQuery denotes that the query should be executed on the entity's labels
 	LabelQuery CriterionType = "labelQuery"
+	// ResultQuery is used to further process result
+	ResultQuery CriterionType = "resultQuery"
```

Comment:
should this be in the `supportedQueryTypes`?

Reply:
Currently no, because it will not be parsed from the URL's query